### PR TITLE
[Feature #159313845] Implement graphql calls

### PR DESCRIPTION
**Pull request by malmike**
#### What Does This PR Do?
- Adds calls to the graphql end points 
- Stores the response data in the redux store

#### Description Of Task To Be Completed
Adds calls to the graphql end points and stores the response data in the redux store

#### Any Background Context You Want To Provide?
Previously the application was using axios to access data from the Rest API. Since the API is being moved to use graphql, the application will then use Apollo Client to retrieve the data from the API as a redux action and then call a reducer to store the data in the redux store.

#### How can this be manually tested?
One should run the application and then open the app in the browser using `{front-end-url-path}/TestActions` and can check for the changes in the redux store using the redux-dev-tools.

#### What are the relevant pivotal tracker stories?
(#159313845)[https://www.pivotaltracker.com/story/show/159313845]

### CODE REVIEW

**Comment by tosmak16** [comment]
Well done @malmike but please there are lots of code climate issues here that are yet to be fixed.

File: client/actions/graphql/eventGQLActions.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+.catch(error => handleError(error, dispatch));
+
+/**
+ * This commented out code below has a use case
+ * where one can not access events that they have deactivated,
+ * You would use this to view them, not sure how it fits in with where one would need
+ * to get the id of the event
+ */
+
+// export const getEvent = id => dispatch => Client.query(
```
Comment (tosmak16) [inline client/actions/graphql/eventGQLActions.js]: well done @malmike but please check line 25 to 28.

Comment (andela-sjames) [reply client/actions/graphql/eventGQLActions.js]: What's the status here

Comment (malmike) [reply client/actions/graphql/eventGQLActions.js]: Removed them, handling the code climate issues then push

Comment (andela-sjames) [reply client/actions/graphql/eventGQLActions.js]: @malmike is this ready for merge?

**Pull request merged**